addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -457,17 +457,28 @@
                         <field name="expense_line_ids" nolabel="1" widget="many2many" domain="[('state', '=', 'draft')]" options="{'not_delete': True}">
                             <tree>
                                 <field name="date"/>
+                                <field name="product_id"/>
                                 <field name="name"/>
                                 <field name="state" invisible="1"/>
+                                <field name="account_id" groups="account.group_account_user"/>
                                 <field name="analytic_account_id" groups="analytic.group_analytic_accounting"/>
                                 <field name="message_unread" invisible="1"/>
                                 <field name="attachment_number" invisible="1"/>
                                 <field name="attachment_number" string=" "/>
                                 <button name="action_get_attachment_view" string="View Attachments" type="object" icon="fa-paperclip"/>
                                 <field name="tax_ids" widget="many2many_tags"/>
-                                <field name="total_amount" sum="Total Amount" widget="monetary"/>
-                           </tree>
+                                <field name="currency_id" invisible="1"/>
+                                <field name="company_currency_id" invisible="1"/>
+                                <field name="total_amount" groups="base.group_multi_currency"/>
+                                <field name="total_amount_company_currency" sum="Total in Company Currency"/>
+                            </tree>
                         </field>
+                        <group class="oe_subtotal_footer oe_right" colspan="2" name="sale_total">
+                                <div class="oe_subtotal_footer_separator oe_inline o_td_label">
+                                    <label for="total_amount_company_currency" />
+                                </div>
+                                <field name="total_amount_company_currency" nolabel="1" class="oe_subtotal_footer_separator"/>
```

Comment:
improve indentation. in subtotal footer field name should "total" enough instead of "total in comany currency".